### PR TITLE
Repair description

### PR DESCRIPTION
**Pull request by ZP-AlwaysWin**
**What type of PR is this?**

> /kind cleanup

**What this PR does / why we need it**:
Repair description

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/kubelet/dockershim/docker_checkpoint.go [hunk 1]
```diff
@@ -61,9 +61,9 @@ type CheckpointData struct {
 type PodSandboxCheckpoint struct {
 	// Version of the pod sandbox checkpoint schema.
 	Version string `json:"version"`
-	// Pod name of the sandbox. Same as the pod name in the PodSpec.
+	// Pod name of the sandbox. Same as the pod name in the Pod ObjectMeta.
```
Comment (mattjmcnaughton) [inline pkg/kubelet/dockershim/docker_checkpoint.go:64]: I'm also seeing similar wording in https://sourcegraph.com/github.com/kubernetes/kubernetes@master/-/blob/staging/src/k8s.io/cri-api/pkg/apis/runtime/v1alpha2/api.proto#L308. 

Can you please share more context around this update? Is it only pertinent here or also in the file I linked above?

**Review by mattjmcnaughton — COMMENTED** [review]
Thanks for your pr!

File: pkg/kubelet/dockershim/docker_checkpoint.go (hunk 1 above)
Comment (ZP-AlwaysWin) [reply pkg/kubelet/dockershim/docker_checkpoint.go:64]: hi @mattjmcnaughton Thanks for your codereview

The Pod structure is defined as follows

```
// Pod is a collection of containers that can run on a host. This resource is created
// by clients and scheduled onto hosts.
type Pod struct {
	metav1.TypeMeta `json:",inline"`
	// Standard object's metadata.
	// More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
	// +optional
	metav1.ObjectMeta `json:"metadata,omitempty" protobuf:"bytes,1,opt,name=metadata"`

	// Specification of the desired behavior of the pod.
	// More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#spec-and-status
	// +optional
	Spec PodSpec `json:"spec,omitempty" protobuf:"bytes,2,opt,name=spec"`

	// Most recently observed status of the pod.
	// This data may not be up to date.
	// Populated by the system.
	// Read-only.
	// More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#spec-and-status
	// +optional
	Status PodStatus `json:"status,omitempty" protobuf:"bytes,3,opt,name=status"`
}
``` 

Pod name and namespace are defined in ObjectMeta, not in PodSpec

Comment (derekwaynecarr) [reply pkg/kubelet/dockershim/docker_checkpoint.go:64]: in this context, it is accurate that this references the Pod name/namespace on the API resource itself.

**Review by derekwaynecarr — APPROVED** [review]
/approve
/kind cleanup

File: pkg/kubelet/dockershim/docker_checkpoint.go (hunk 1 above)
Comment (ZP-AlwaysWin) [reply pkg/kubelet/dockershim/docker_checkpoint.go:64]: > in this context, it is accurate that this references the Pod name/namespace on the API resource itself.

@derekwaynecarr Thanks your code review.
Please help to tag / lgtm

**Comment by ZP-AlwaysWin** [comment]
> Thanks for your pr!

Can you help me label / lgtm

**Pull request merged**